test/parallel.jl

```diff
@@ -223,12 +223,16 @@ if Bool(parse(Int,(get(ENV, "JULIA_TESTFULL", "0"))))
 
     # retry, on error exit
     res = pmap(x->(x=='a') ? error("EXPECTED TEST ERROR. TO BE IGNORED.") : uppercase(x), s; err_retry=true, err_stop=true);
-    @test length(res) < length(ups)
+    if length(res) == length(ups)
+        println("WARNING! Everything finished processing in spite of an error. Though possible it is a typically unexpected condition!")
```

Comment:
Why not just `warn` ? "Though possible it is a typically unexpected condition" sounds a little funny, doesn't the fact that this is a warning message already convey that?